ci/qa/jscpd

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env sh
+
+cd $(dirname $0)/../../
+
+# Run the JS CPD tool:
+# - allowing for 30 lines of duplicate code
+# - and setting a total copy paste limit to 1% project wide
+# More info about jscpd usage: https://github.com/kucherenko/jscpd/tree/master/packages/jscpd#usage
+./node_modules/.bin/jscpd src  -l 30 -t 1
```

Comment:
Did you not end up adding the excluded folders? If this command does not give too many false positives, I'ts fine with me. I do think we need to both check the JS and PHP code though. And the JS code is not in the `src` folder IIRC.